index.html

```diff
@@ -11,12 +11,12 @@
         <link href="https://fonts.googleapis.com/css?family=Roboto%7CRoboto+Condensed" rel="stylesheet">
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.1/dist/katex.min.css" integrity="sha384-dbVIfZGuN1Yq7/1Ocstc1lUEm+AT+/rCkibIcC/OmWo5f0EA48Vf8CytHzGrSwbQ" crossorigin="anonymous">
         <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
-        <link rel="icon" type="image/png" href="/dist/images/favicon-16x16.png">
+        <link rel="icon" type="image/png" href="/src/images/favicon-16x16.png">
         
     </head>
     
     <body>
-        <div class="sr-only">
+        <div className="sr-only">
```

Comment:
@mrcrymnd `className` is only needed in `*.jsx` files, 'class' is a reserved keyword in Javascript